docs/shared/recipes/repo-types/integrated-in-package-based.md

```diff
@@ -19,7 +19,7 @@ Even if the project you added only uses javascript, a `tsconfig.base.json` file
 
 ## project.json
```

Comment:
I wonder whether this page is needed at all. I have a hard time to think about who would go and read this specific one, also because with Crystal you'd expect to be able to use `nx add @nx/react` and then keep going (which we're not 100% there yet especially with generators).

This page also still contains steps for first `npm install` etc.. rather than using `nx add`.

Reply:
I can definitely amend the `npm install` but this PR is for a task to address some `targetDefaults` examples, not to update the full pages. Whether the page is still needed or not, that's also a separate conversation. I don't think we're there yet. Generators definitely don't support package-based repos yet.